src/schema/ecommerce/set_order_payment_mutation.ts

```diff
@@ -54,26 +54,27 @@ export const SetOrderPaymentMutation = mutationWithClientMutationId({
             __typename
             ... on EcommerceOrderWithMutationSuccess {
               order {
-              id
-                code
-                currencyCode
-                state
+                id
```

Comment:
this looks more complicated than what it is, i basically ordered fields and only added `creditCardId`.